packages/experiments-realm/productivity/task.gts

```diff
@@ -329,9 +347,8 @@ class Fitted extends Component<typeof Task> {
         padding: var(--boxel-sp-sm);
         display: flex;
         flex-direction: column;
-        justify-content: space-between;
         gap: var(--boxel-sp-sm);
-        overflow: auto;
+        overflow-y: auto;
```

Comment:
is this overflow property necessary? it would add a third layer of scroll on the task app. instead you can say `overflow: hidden` and make sure all the inner items have ellipsis or other ways to handle the extra content.